Module1/todo_app/test_app.py

```diff
@@ -0,0 +1,57 @@
+from dotenv import load_dotenv, find_dotenv
+import os
+from flask.testing import FlaskClient
+from todo_app import app
+import pytest
+import requests
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create the new app.
+    test_app = app.create_app()
+
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+class StubResponse():
+    def __init__(self, fake_response_data):
+        self.fake_response_data = fake_response_data
+
+    def json(self):
+        return self.fake_response_data
+
+def stub(url, params={}, headers=None):
+    test_board_id = os.environ.get('BOARDID')
+    print(test_board_id)
+    fake_response_data = None
+
+    if url == f'https://api.trello.com/1/boards/{test_board_id}/cards':
+        fake_response_data = [{
+            'name': 'To Do',
+            'DESC': '',
+            'ID': '123abc',
```

Comment:
This needs to match the casing that _Trello_ uses rather than our app, since we're faking the responses from Trello. So I think these should both be lowercase, and `listId` for that